api/hooks/issues-hook/update-work-item-priority/index.js

```diff
@@ -6,15 +6,23 @@
  */
 
 const Issues = require('../../../services/Issues');
+const logger = require('../../../services/Logs/logger');
 
 module.exports = async function updateWorkItemPriority({
     priority,
     relatedUrl
 }) {
     const issue = await Issues.getByRelatedUrl(relatedUrl);
+
+    // TODO bug? is "priority" comparable with '>' ? The string "P2" is greater than "P1" but semantically its the other way around
```

Comment:
Is this related to #100? Also, we should probably use issues instead of leaving TODO comments.

Reply:
Unrelated, this is a bug discovered that made me update the test above. Agreed, opened #150 for this.